lib/style.css

```diff
@@ -56,6 +56,20 @@ body {
   background-color: <<BACK_PRIMARY>>;
 }
 
+#player.material paper-icon-button.playing::after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-top: -1.2em;
+  height: 2.4em;
+  width: 2.4em;
```

Comment:
Fwiw, we could grab dimensions via JS but I dislike that as it feels even more like a hack